cmake/developer_package/api_validator/api_validator.cmake

```diff
@@ -88,9 +88,12 @@ function(_ie_add_api_validator_post_build_step)
 
     macro(api_validator_get_target_name)
         get_target_property(IS_IMPORTED ${target} IMPORTED)
+        get_target_property(orig_target ${target} ALIASED_TARGET)
```

Comment:
Why do we need to have these changes?

Reply:
That is a fix by @ilya-lavrenov. Better ask him

Reply:
it's already a part of https://github.com/openvinotoolkit/openvino/pull/6508

Looks like after merge with current master we should not see these changes